packages/components/package.json

```diff
@@ -107,13 +106,17 @@
     "@kaizen/design-tokens": "workspace:*",
     "@kaizen/package-bundler": "workspace:*",
     "@tanstack/react-query": "^5.51.23",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.5.0",
+    "@types/jest-axe": "^3.5.9",
     "@types/lodash.debounce": "^4.0.9",
     "@types/react-highlight": "^0.12.8",
     "@types/react-textfit": "^1.1.4",
     "@types/uuid": "^10.0.0",
     "autoprefixer": "^10.4.20",
     "concat-cli": "^4.0.0",
     "identity-obj-proxy": "^3.0.0",
+    "jest-axe": "^9.0.0",
```

Comment:
Do we still need this or were you able to fix all the tests?

Reply:
We still need it for Storybook's test-runner... But admittedly shouldn't be in this `package.json`